features/step_definitions/collection_steps.rb

```diff
@@ -122,6 +122,39 @@
   visit collections_path
 end
 
+Given "a set of collections for searching" do
+  profile = CollectionProfile.create!(faq: "<dl><dt>What is this test thing?</dt><dd>It's a test collection</dd></dl>",
+                                      intro: "Welcome to the test collection",
+                                      rules: "Be nice to testers")
+  FactoryBot.create(:collection,
+                    name: "sometest",
+                    title: "Some Test Collection",
+                    collection_profile: profile)
+  FactoryBot.create(:collection,
+                    name: "othertest",
+                    title: "Some Other Collection")
+  FactoryBot.create(:collection,
+                    name: "anothertest",
+                    title: "Another Plain Collection",
+                    collection_preference: CollectionPreference.create!(closed: true))
```

Comment:
Do you think it would be helpful to have a factory or trait for this and moderated?